unlock-app/src/config/queryClient.ts

```diff
@@ -6,37 +6,36 @@ import * as Sentry from '@sentry/nextjs'
 export const queryClient = new QueryClient({
   queryCache: new QueryCache({
     onError: (error: any, query) => {
+      if (
+        error instanceof AxiosError &&
+        error?.response?.status &&
+        [401, 403, 404].includes(error.response.status)
+      ) {
+        return
+      }
+
       const id = Sentry.captureException(error, {
         contexts: {
           query: {
             queryKey: query.queryKey,
           },
         },
       })
+
       console.debug(`Event ID: ${id}\n`, error)
-      if (
-        error instanceof AxiosError &&
-        error?.response?.status &&
-        [404, 401, 403].includes(error.response.status)
-      ) {
-        return
-      }
+
       if (query?.meta?.errorMessage) {
         toast.error(query.meta.errorMessage as string)
       } else {
         switch (error?.code) {
           case -32000:
           case 4001:
           case 'ACTION_REJECTED':
-            toast.error('Transaction rejected.')
+            toast.error('Transaction rejected')
             break
           default: {
             const errorMessage = error?.error?.message || error.message
-            if (errorMessage) {
-              toast.error(
-                'There was an unexpected error. Please try again, refresh the page and report to our team if this persists.'
-              )
-            }
+            console.error(errorMessage)
```

Comment:
Let's keep a user readable mesasge here because too often the `errorMessage` we get are not legible.